src/components/InputBook.js

```diff
@@ -18,29 +20,33 @@ export const InputBook = () => {
     });
   };
 
-  const handleAddBook = (e) => {
+  const handleAddBook = async (e) => {
     e.preventDefault();
     if (inputBook.title.trim()) {
+      const bookObj = {
+        item_id: uuidv4(),
+        title: inputBook.title,
+        author: inputBook.author,
+        category: inputBook.category,
+      };
+
+      await dispatch(addBookEvent(bookObj));
+      await dispatch(LoadBooks());
       setInputBook({
         title: '',
         author: '',
+        category: '',
       });
-      dispatch(
-        addBookEvent({
-          title: inputBook.title,
-          author: inputBook.author,
-        }),
-      );
     } else {
       // eslint-disable-next-line no-alert
       alert('Please write a title');
     }
   };
 
   return (
-    <div>
+    <div className="div-form">
+      <h2 className="form-header"> Add a New Book</h2>
```

Comment:
- [ ] Please implement the horizontal border before the form as shown in the zeplin design
- [ ] Please increase the width of the form div so that it is well aligned with the other content on the page. I have attached a diagram for more information
- [ ] Please align the Add a New Book heading to the left as shown in the given design
### Your Design
![image](https://user-images.githubusercontent.com/46839548/208251701-95173c48-2820-4776-b7f9-bba4930e004f.png)

### Zeplin Design
![image](https://user-images.githubusercontent.com/46839548/208251737-c38ce02e-4294-4cf7-9dd4-cebea197af90.png)